src/release/extensionApi.js

```diff
@@ -217,38 +183,34 @@ module.exports = function extensionApiFactory(params = {}, data = { extension: {
          * @param {String} releasingBranch
          */
         async updateTranslations(releasingBranch) {
-            log.doing('Translations');
-
-            // Start with CLI option, if it's missing we'll prompt user
-            let translation = params.updateTranslations;
-
-            if (!translation) {
+            //translations runs only in interactive mode
+            if (interactive && updateTranslations) {
```

Comment:
If you check the old version, we were checking `if (!translation)` to launch the prompt. And then checking again to begin the translation. The new version is not equivalent.